cmake/CMakeLists.txt

```diff
@@ -716,6 +716,17 @@ else()
   endif()
 endif()
 
+if(onnxruntime_USE_KLEIDIAI)
+  # If KLEIDIAI_DEBUG is enabled that implies both DEBUG and KERNEL messages.
+  if(onnxruntime_KLEIDIAI_DEBUG_LOGGING)
+    target_compile_definitions(onnxruntime_mlas PRIVATE -DKLEIDIAI_DEBUG=1)
+    target_compile_definitions(onnxruntime_mlas PRIVATE -DKLEIDIAI_KERNEL=1)
+  endif()
+  if(onnxruntime_KLEIDIAI_KERNEL_LOGGING)
+    target_compile_definitions(onnxruntime_mlas PRIVATE -DKLEIDIAI_KERNEL=1)
```

Comment:
I believe this will be run before the onnxruntime_mlas target has been created in onnxruntime_mlas.cmake which is included below.

maybe it would be simpler to move this code to onnxruntime_mlas.cmake.